edge/poll.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 const tunnelActivityCheckInterval = 30 * time.Second
+const waitToBeAssociated = 0
```

Comment:
The name is not accurate, the `0` value means that the agent is using a global edge key so the endpoint ID will be automatically created, independently of the waiting room.

Reply:
Do you have any suggestions for a better name? '0' doesn't really convey any meaning and could lead to confusion.

Reply:
Have renamed it `globalKeyInUse`